lib/doorkeeper/oauth/pre_authorization.rb

```diff
@@ -35,7 +35,7 @@ def scopes
       end
 
       def scope
-        @scope.presence || build_scopes
+        @scope.presence || (server.default_scopes.presence && build_scopes)
```

Comment:
Actually, `build_scopes` returns `server.default_scopes.to_s`. So maybe we can just remove `#to_s` from it and do it on other place after we make sure scopes present?

Reply:
I think we don't have to change this because current code  makes sure scopes present. 
`@scope` is set by request's `params[:scope]` by this line: https://github.com/doorkeeper-gem/doorkeeper/blob/62a304dfb6b31df0201a1b29ac3a568004dbc24d/lib/doorkeeper/oauth/pre_authorization.rb#L23 so it's type is `string`, 
`build_scopes` also be `string` by https://github.com/doorkeeper-gem/doorkeeper/blob/62a304dfb6b31df0201a1b29ac3a568004dbc24d/lib/doorkeeper/oauth/scopes.rb#L41-L43
Then rails will check string empty by `blank?` method.

Reply:
OK, it's ok for me